src/material-experimental/mdc-form-field/form-field.ts

```diff
@@ -558,10 +556,17 @@ export class MatFormField
     if (!this._hasOutline() || !this._floatingLabel) {
       return;
     }
+    if (!this._labelScale) {
+      this._labelScale = Number(
+        getComputedStyle(this._elementRef.nativeElement).getPropertyValue(
```

Comment:
Should we cache this once for the entire app, rather than having to check it for each form field?

Reply:
I went with Paul's suggestion which makes this unnecessary